PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1461,11 +1461,11 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	FindOption env;
 	if (isBlankContained)
 	{
-		env._str2Search = TEXT("^[\\t ]*$(\\r\\n|\\r|\\n)");
+		env._str2Search = TEXT("^([\\t ]*$(\\r?\\n|\\r))(\\1)*");
 	}
 	else
 	{
-		env._str2Search = TEXT("^$(\\r\\n|\\r|\\n)");
+		env._str2Search = TEXT("^$(\\r?\\n|\\r)(\\1)*");

```

Comment:
Isn't this a better regex here?: `^\R+`

Reply:
As above if necessary this will also be simpler `^(\\r?\\n|\\r)+`.